lib/pdk/module/metadata.rb

```diff
@@ -1,3 +1,4 @@
+require 'pdk/util'
```

Comment:
We're trying to keep the number of top level requires down to a minimum, can you please move this down to line 111 so that it is lazy loaded in `PDK::Module::Metadata.from_file` only when it is needed?